launch-entrypoint.sh

```diff
@@ -1,25 +1,61 @@
 #!/usr/bin/env bash
 
-box()
-{
-  local s=("$@") b w
-  for l in "${s[@]}"; do
-    ((w<${#l})) && { b="$l"; w="${#l}"; }
+set -euxo pipefail
+
+main() {
+
+  trap show_exit_warning EXIT
+  trap show_exit_warning SIGTERM
+
+  fix_aws_environment_variables
+  write_bash_aliases
+  write_inputrc
+
+  exec bash
+}
+
+fix_aws_environment_variables() {
+  export AWS_REGION="${AWS_REGION:-${AWS_DEFAULT_REGION:-}}"
+}
+
+write_bash_aliases() {
+  cat >>/home/launch/.bash_aliases <<EOF
+alias ll='ls -lasp'
+EOF
```

Comment:
I think we should just include a static .bash_aliases file rather than write it every time we run the container

Reply:
Same for inputrc

Reply:
Also the `>>` means the aliases will grow duplicates on each run